frontend/src/components/profile/components/profile-info/profile-info.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { IUser } from 'common/interfaces/user';
+import { getDateFormat } from 'helpers';
+import clsx from "clsx";
+import styles from './styles.module.scss';
+
+type Props = {
+  user: IUser | null
+  edit: () => void
+}
+
+const ProfileInfo: React.FC<Props> = ({user, edit}) => {
+  let birthdate
+  if (user) birthdate = getDateFormat(user?.birthdate)
```

Comment:
Create helper for it.
Something like `const getFormattedDate = (date, formattedDate: FormattedDateType) => {}`